miden/tests/integration/cli/cli_test.rs

```diff
@@ -0,0 +1,12 @@
+use crate::{build_test};
+
+#[test]
+fn cli_run() {
+    let cli = Cli::from_args();
+    let source = "begin repeat.1000 swap dup.1 add end end";
+    let pub_inputs = vec![1, 1];
+
+    // This does not test the RunCmd and the outputs yet. 
+    // I need some input on how to best do that.
+    build_test!(source, &pub_inputs).execute(pub_inputs, false);
+}
```

Comment:
Here I need some help how we could add CLI tests in general. We could add a function to build a CLI in the macro `built_test!` and then look at the output?